radixdlt-java/src/main/java/com/radixdlt/client/core/network/RadixNode.java

```diff
@@ -0,0 +1,54 @@
+package com.radixdlt.client.core.network;
+
+import okhttp3.Request;
+
+/**
+ * Unique network identifier for a node.
+ */
+public class RadixNode {
+	private final String location;
+	private final boolean useSSL;
+	private final int port;
+	private final Request request;
+	private final String url;
+
+	public RadixNode(String location, boolean useSSL, int port) {
+		this.location = location;
+		this.useSSL = useSSL;
+		this.port = port;
+		this.url = (useSSL ? "wss://" : "ws://") + location + ":" + port + "/rpc";
+		this.request = new Request.Builder().url(url).build();
+	}
+
+	public Request getLocation() {
+		return request;
+	}
+
+	public int getPort() {
+		return port;
+	}
+
+	public boolean isSsl() {
+		return useSSL;
+	}
+
+	@Override
+	public String toString() {
+		return location;
+	}
+
+	@Override
+	public int hashCode() {
+		return url.hashCode();
+	}
+
+	@Override
+	public boolean equals(Object o) {
+		if (!(o instanceof RadixNode)) {
+			return false;
+		}
+
+		RadixNode p = (RadixNode) o;
+		return p.url.equals(this.url);
```

Comment:
Okay, maybe the description is not entirely accurate as a "Unique" Radix node *really* should mean a node associated with a key so this class should really be `RadixNodeEndpoint` which represents a unique URL (aka it could be the same machine).

Down the line though, this needs to be replaced with the node key and it's known associated endpoints. I don't think we're there yet though.